libs/community/tests/integration_tests/document_loaders/test_unstructured.py

```diff
@@ -87,7 +95,7 @@ def test_unstructured_api_file_io_loader() -> None:
             api_key="FAKE_API_KEY",
             strategy="fast",
             mode="elements",
-            file_filename=file_path,
```

Comment:
The loaders used this `file_filename` parameter I haven't seen before and don't see in the documentation ([here](https://python.langchain.com/v0.2/docs/integrations/providers/unstructured/) or [here](https://api.python.langchain.com/en/latest/document_loaders/langchain_community.document_loaders.unstructured.UnstructuredAPIFileLoader.html)). I assume I need to add logic so this param is still accepted, but please let me know if there's anything else to consider in how that should be handled. I replaced it with `metadata_filename` since that's the param we use in `unstructured`, but that's not actually a recognized [param](https://docs.unstructured.io/api-reference/api-services/api-parameters) for the sdk either.

Reply:
You're good to drop `file_filename`. `file_filename` was deprecated a while back in favor of `metadata_filename`.

https://github.com/Unstructured-IO/unstructured/pull/1191